packages/orchestration/src/exos/orchestrator.js

```diff
@@ -119,17 +129,31 @@ export const prepareOrchestratorKit = (
             return watch(
               chainHub.getChainInfo('agoric'),
               this.facets.makeLocalChainFacadeWatcher,
+              name,
             );
           }
           return watch(
             chainHub.getChainsAndConnection('agoric', name),
             this.facets.makeRemoteChainFacadeWatcher,
+            name,
           );
         },
         makeLocalAccount() {
           return watch(E(localchain).makeAccount());
         },
-        getBrandInfo: () => Fail`not yet implemented`,
+        /** @type {HostOf<Orchestrator['getBrandInfo']>} */
+        getBrandInfo(denom) {
+          const { chainName, baseName, baseDenom, brand } =
+            chainHub.lookupAsset(denom);
+          chainByName.has(chainName) ||
+            Fail`use getChain(${q(chainName)}) before getBrandInfo(${q(denom)})`;
```

Comment:
this is surprising. `getFoo` sounds idempotent. Is there away around this? If not, shall we rename `getChain`?

Reply:
It's part of the "you have to tell it all the answers before you ask the question" design that lets `getBrandInfo()` be sync.

I could go with renaming `getChain` -> `provideChain`, provided the keepers of `orchestration-api.d.ts` agree.

Reply:
Ok, out of scope